definitions/types/aws-security.json

```diff
@@ -4,43 +4,13 @@
   "title": "AWS Security information",
   "description": "Informs downstream services of ports to open for communication",
   "additionalProperties": false,
-  "required": [
-    "groups"
-  ],
+  "required": [],
   "properties": {
     "groups": {
```

Comment:
Leaving this in for backwards compatibility, but planning on removing it from the AWS bundles

Reply:
Nice, I like the removal of the `required` but leaving the key here for backwards-compat.